packages/common/src/api/suggestedTracks.ts

```diff
@@ -79,34 +82,38 @@ export const useGetSuggestedAlbumTracks = (collectionId: ID) => {
     { disabled: !currentUserId }
   )
 
-  useCustomCompareEffect(
-    () => {
-      if (status === Status.SUCCESS && ownTracks) {
-        const suggestedTrackIds = difference(
-          shuffle(ownTracks).map((track) => track.track_id),
-          collectionTrackIds
-        )
-        setSuggestedTrackIds(suggestedTrackIds)
-      }
-      // eslint-disable-next-line react-hooks/exhaustive-deps
-    },
-    [status, ownTracks?.length],
-    isEqual
-  )
+  const reset = useCallback(() => {
```

Comment:
i wonder if there is a better name for this function, but can't think of anything great, like "shuffle" or update, or showMore?